services/svc/digitaltwins/src/package_2021_06_30_preview/operations.rs

```diff
@@ -16,7 +16,7 @@ pub struct ClientBuilder {
     endpoint: Option<String>,
     scopes: Option<Vec<String>>,
 }
-pub const DEFAULT_ENDPOINT: &str = azure_core::resource_manager_endpoint::AZURE_PUBLIC_CLOUD;
+pub const DEFAULT_ENDPOINT: &str = "https://digitaltwins-hostname";
```

Comment:
Not needed for this PR, as the previous version was wrong for this crate as well.

1. dotnet: `this.endpoint = endpoint ?? new Uri("https://digitaltwins-name.digitaltwins.azure.net");`
2. python: required to specify as an argument
3. golang: `DefaultBaseURI = "https://digitaltwins-name.digitaltwins.azure.net"`

Reply:
NOTE: As of today (2021-01-10), `digitaltwins-name.digitaltwins.azure.net` doesn't resolve.  That means the dotnet and golang version are broken unless the user specifies an alternate endpoint, which are considered optional in the API.